packages/dd-trace/src/appsec/index.js

```diff
@@ -90,18 +102,21 @@ function incomingHttpStartTranslator ({ req, res, abortController }) {
     payload[addresses.HTTP_CLIENT_IP] = clientIp
   }
 
+  if (sampleRequest(config.appsec.apiSecurity)) {
+    payload[addresses.WAF_CONTEXT_PROCESSOR] = { 'extract-schema': true }
+  }
+
   const actions = waf.run(payload, req)
 
   handleResults(actions, req, res, rootSpan, abortController)
 }
 
 function incomingHttpEndTranslator ({ req, res }) {
-  // TODO: this doesn't support headers sent with res.writeHead()
```

Comment:
Why did you remove this comment ?

Reply:
Probably not the TODO comment I wanted to delete